assets/rln-relay/rln-relay.msc

```diff
@@ -0,0 +1,33 @@
+# MSC for some fictional process
```

Comment:
? :D

Reply:
(^_^;) left from a sample msc code, forgot to erase